src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -58,40 +61,30 @@
 
     <!--Used to create a header for a control-->
     <Style x:Key="CustomSettingHeaderStyle" TargetType="TextBlock">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="Margin" Value="0,0,0,4"/>
     </Style>
 
-    <!-- Used to create a header for a group of settings -->
-    <Style x:Key="GroupingHeader" TargetType="TextBlock" BasedOn="{StaticResource SubtitleTextBlockStyle}">
-        <Setter Property="Margin" Value="0,20,0,10"/>
-    </Style>
-
     <!--Used for disclaimers-->
     <Style x:Key="DisclaimerStyle" TargetType="TextBlock">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="FontStyle" Value="Italic"/>
         <Setter Property="TextWrapping" Value="WrapWholeWords"/>
     </Style>
 
     <!--Used for flyout messages-->
     <Style x:Key="CustomFlyoutTextStyle" TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="FontWeight" Value="Bold"/>
         <Setter Property="Margin" Value="0,0,0,12"/>

```

Comment:
Mind checking if this margin is correct too? (low priority)

Reply:
Yep, this is for the spacing between the text and the button in the flyout.